rstudio/rstudio
# Add commands for showing each Global Options pane directly

### Intent

Provide ability to go directly to each pane in the Global Options dialog from the command palette.

Fixes #7678

### Approach

Added new commands for panes that didn't already have one (terminal, accessibility, and pane layout already had such commands). The command are:

- Global Options (goes to General pane)
- Code Options...
- Console Options...
- Appearance Options...
- Pane Layout Options...
- Packages Options...
- R Markdown Options...
- Sweave Options...
- Spelling Options...
- Git/SVN Version Control Options...
- Publishing Options...
- Terminal Options...
- Accessibility Options... (note this is a special case as it opens JUST the accessibility pane, no other panes available, this is on purpose to reduce potential confusion for screen reader users)
- Python Options...

### QA Notes

Verify that each commands opens the corresponding pane in Global Options.

Note that three of these panes can be hidden via feature limits in RSP (`allow_shell` for terminal, `allow_publish` for publishing, `allow-vcs` for version control); ensure that when such limits are set, the corresponding command is not shown in the command palette.

Also fixed a tiny issue that was introduced via #3564; if Terminal is disallowed via the "allow_shell" feature limit (RSP) the "Close all Terminals" was not being removed, causing the Tools / Terminal menu to still appear, so now hiding it
